src/mds/topology/topology_storge_etcd.cpp

```diff
@@ -82,7 +82,7 @@ bool TopologyStorageEtcd::LoadPhysicalPool(
         LOG(ERROR) << "etcd list err:" << errCode;
         return false;
     }
-    for (int i = 0; i < out.size(); i++) {
+    for (int i = 0; i < static_cast<int>(out.size()); i++) {
```

Comment:
```suggestion
    for (size_t i = 0; i < out.size(); i++) {
```

and the following for-loop also needs to be modified in this way